iModelCore/libsrc/libxml2/BeLibxml-unix-config.h

```diff
@@ -337,3 +337,8 @@
 
 /* Include multi-threading support like win32 already does. This triggers more in xmlversion.h. */
 #define __MT__
+
+/* Include XML_SYSCONFDIR for unix. As of 2.14.4 this Macro is no longer defined in libxml.h */
+#ifndef _MS_VER

```

Comment:
```suggestion
#ifndef XML_SYSCONFDIR
```

Reply:
That way if it ever gets defined somewhere else (like a makefile), this will leave it alone.